content/docs/rfcs/36/README.md

```diff
@@ -0,0 +1,612 @@
+      Default `20`.
+    - `relay` - `boolean` (optional): Enable relay protocol.
+      Default `true`.
+    - `minPeersToPublish` - `number` (optional).
+      The minimum number of peers required on a topic to allow broadcasting a message.
+      Default `0`.
+
+**Returns**
+
+A [`JsonResponse`](#jsonresponse).
+If the execution is successful, the `result` field contains an `integer` which represents the `nodeId`.
+This node id MUST be used in following API calls to interact with the instantiated node.
+
+For example:
+
+```json
+{
+  "result": 0
+}
+```
+
+### `extern char* waku_start(int nodeId)`
+
+Start a Waku node mounting all the protocols that were enabled during the Waku node instantiation.
+
+**Parameters**
+
+1. `int nodeId`: The node identifier obtained from a successful execution of [`waku_new`](#extern-char-waku_newchar-configjson).
+
+**Returns**
+
+A [`JsonResponse`](#jsonresponse).
+If the execution is successful, the `result` field is set to `null`.
+
+For example:
+
+```json
+{
+  "result": null
+}
+```
+
+### `extern char* waku_stop(int nodeID)`
+
+Stops a Waku node.
+
+**Parameters**
+
+1. `int nodeId`: The node identifier obtained from a successful execution of [`waku_new`](#extern-char-waku_newchar-configjson).
+
+**Returns**
+
+A [`JsonResponse`](#jsonresponse).
+If the execution is successful, the `result` field is set to `null`.
+
+For example:
+
+```json
+{
+  "result": null
+}
+```
+
+### `extern char* waku_peerid(int nodeId)`
+
+Get the peer ID of the waku node.
+
+**Parameters**
+
+1. `int nodeId`: The node identifier obtained from a successful execution of [`waku_new`](#extern-char-waku_newchar-configjson).
+
+**Returns**
+
+A [`JsonResponse`](#jsonresponse).
+If the execution is successful, the `result` field contains the peer ID as a `string` (base58 encoded).
+
+For example:
+
+```json
+{
+  "result": "QmWjHKUrXDHPCwoWXpUZ77E8o6UbAoTTZwf1AD1tDC4KNP"
+}
+```
+
+### `extern char* waku_listen_addresses(int nodeId)`
+
+Get the multiaddresses the Waku node is listening to.
+
+**Parameters**
+
+1. `int nodeId`: The node identifier obtained from a successful execution of [`waku_new`](#extern-char-waku_newchar-configjson).
+
+**Returns**
+
+A [`JsonResponse`](#jsonresponse).
+If the execution is successful, the `result` field contains an array of multiaddresses.
+The multiaddresses are `string`s.
+
+For example:
+
+```json
+{
+  "result": [
+    "/ip4/127.0.0.1/tcp/30303",
+    "/ip4/1.2.3.4/tcp/30303",
+    "/dns4/waku.node.example/tcp/8000/wss"
+  ]
+}
+```
+
+## Connecting to peers
+
+### `extern char* waku_add_peer(int nodeId, char* address, char* protocolId)`
+
+Add a node multiaddress and protocol to the waku node's peerstore.
+
+**Parameters**
+
+1. `int nodeId`: The node identifier obtained from a successful execution of [`waku_new`](#extern-char-waku_newchar-configjson).
+2. `char* address`: A multiaddress to reach the peer being added.
```

Comment:
Multiaddr format:
- multiaddr with peer id supported?
- multiaddr without peer id supported?

Reply:
The peer id is required